src/client/Microsoft.Identity.Client/ConfidentialClientApplication.cs

```diff
@@ -37,6 +37,11 @@ public sealed partial class ConfidentialClientApplication
             IConfidentialClientApplicationWithCertificate,
             IByRefreshToken
     {
+        /// <summary>
+        /// Instructs MSAL to try to auto discover the Azure region.
+        /// </summary>
+        public const string AttemptRegionDiscovery = "TryAutoDetect";
```

Comment:
Why is this public if it is a const string?

Reply:
Oh I see, this is used as a optional parameter in the new api 

---
In reply to: [602606370](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/2509#discussion_r602606370) [](ancestors = 602606370)